SSSwiftUIAnimations/Sources/WaterProgressAnimation/CheckmarkView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CheckmarkView.swift
+//  SSSwiftUIAnimations
+//
+//  Created by Rahul Yadav on 15/07/24.
+//
+
+import SwiftUI
+
+struct CheckmarkView: View {
+    
+    @State var displayCheckmark: Bool = false
+    var style: SSWaterProgressViewStyle
+    @State private var zoom = 0.1
+    
+    var body: some View {
+        ZStack {
+            // Checkmark
```

Comment:
Add tint color of CheckmarkView or custom image from the user if they want to add otherwise give default params